app/javascript/components/widgets/twitter/tweet_content.jsx

```diff
@@ -86,10 +86,10 @@ TweetText.propTypes = {
 };
 
 const imageLayout = images => {
-  return images.map(image => {
+  return images.map((image, index) => {
     const isMain = image === images[0] && images.length === 3;
     return (
-      <ImageWrapper main={isMain}>
+      <ImageWrapper key={`${image}-${index}`} main={isMain}>
```

Comment:
this is failing a linter check
> /home/circleci/repo/app/javascript/components/widgets/twitter/tweet_content.jsx
>  92:26  error  Do not use Array index in keys  react/no-array-index-key

Im okay if you disable this check, I don't think its valuable.

or `key={image}` would probably also work fine

Reply:
Okay, how would I disable the check?

Reply:
I wouldn't disable the check, it's there for a reason.

React uses the `key` to be able to tell which child is which in the event that a list is reordered or has an item inserted or removed. That allows it to reorder the existing children in the DOM rather than removing them all and recreating them in the correct order. However, changing the list will also change the indices of the items in the list, which will in turn change their keys. That means React won't be able to pair the old and new keys and will still have to drop and recreate the child.

A key with an array index in it is logically equivalent to omitting the key. If you're really okay omitting the key then silence _that_ error, otherwise pick a key that doesn't include the index.